<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdBox</class>
 <widget class="QDialog" name="CmdBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CmdBox</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QLineEdit" name="cmdEdit">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="cmdList">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>cmdList</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>runButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>CmdBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
